components/test/test_subprocess.py

```diff
@@ -156,8 +156,6 @@ def f(
                 raise ValueError
 
             q = 1
-            # Trailing comment isn't part of the execution, so it is actually
```

Comment:
Need to remove the trailing comments because inspect in Python 3.8 fixed the bug of dropping trailing comments.

Reply:
I think this is fine. (It's a very niche case and doesn't affect execution.)

Reply:
Thanks for clarification!